repairnator/repairnator-scanner/src/main/java/fr/inria/spirals/repairnator/scanner/ProjectScanner.java

```diff
@@ -99,15 +99,15 @@ public Date getLimitDate() {
         return limitDate;
     }
 
-    public List<BuildToBeInspected> getListOfFailingBuildsFromGivenBuildIds(String path, LauncherMode mode) throws IOException {
-        return getListOfBuildsFromGivenBuildIds(path, mode, true);
+    public List<BuildToBeInspected> getListOfFailingBuildsFromGivenBuildIds(String path, LauncherMode launcherMode) throws IOException {
```

Comment:
In fact, instead of passing the mode everywhere in the methods: wouldn't it easier to create a field for the mode and pass it in the constructor?